src/js/router/router.js

```diff
@@ -27,14 +28,12 @@ const routes = {
     apiCallBack: (params) => console.log(`your data and ${params}`),
   },
   '/exercises': {
-    route: `${basePath}/exercises.html`,
-    domCallBack: () => console.log('your element'),
-    apiCallBack: (params) => console.log(`your data and ${params}`),
+    route: '/pages/exercises.html',
```

Comment:
Please update main and be carefull
Should be
`route: `${basePath}/exercises.html`,`

Reply:
Ok